src/de/seq.rs

```diff
@@ -96,29 +82,7 @@ where
 {
     /// Creates a new accessor to a top-level sequence of XML elements.
     pub fn new(de: &'a mut Deserializer<'de, R>) -> Result<Self, DeError> {
-        let filter = if let DeEvent::Start(e) = de.peek()? {
-            // Clone is cheap if event borrows from the input
-            TagFilter::Include(e.clone())
-        } else {
-            TagFilter::Exclude(&[])
-        };
-        Ok(Self {
-            #[cfg(feature = "overlapped-lists")]
-            checkpoint: de.skip_checkpoint(),
-
-            de,
-            filter,
-        })
-    }
-}
-
-#[cfg(feature = "overlapped-lists")]
-impl<'de, 'a, R> Drop for TopLevelSeqAccess<'de, 'a, R>
-where
-    R: XmlRead<'de>,
-{
-    fn drop(&mut self) {
-        self.de.start_replay(self.checkpoint);
+        Ok(Self { de })
```

Comment:
Does this need to remain fallible?  ie. can it just return `Self` instead of `Result<Self, DeError>`

Reply:
Yeah, missed that. Will fix in few seconds...